### ISSUE TITLE
All fields and register button set to error state when email already exist

### ISSUE THREAD

**Issue by kuncevic** [AnthonyNahas/ngx-auth-firebaseui#227]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps
Create user with the existing email

In case user all the fields and button just gets coloured red after you click register button and all fields gets blanked.

![image](https://user-images.githubusercontent.com/1283734/56283989-90724380-6156-11e9-83e1-2ba6b566dcfb.png)

### Desired functionality 
Solution 1: just to color the email field red (name, password and button must stay like it is with no error state). It would be great to leave the username text so user do not have to type it again

Solution 2: to use remote validation to check if email exist during user typing.

**Comment by AnthonyNahas** [comment]
@kuncevic thanks for suggesting the above solutions, however the second one would be very difficult to realize since we need access of the firebase admin sdk and firestore access rules...

The first requested solution is easier to implement and we need here to implement a custom validator... 

This will take a bit time (since I have too many project at the moment to release)

I will appreciate your help if you could request a PR inc. the above requirements.


Cheers 🍻

**Comment by kuncevic** [comment]
Yeah i thought so that the Solution 2 might be more hard to implement. I will see what I can do in regards of implementing a PR based on Solution 1.

**Comment by AnthonyNahas** [comment]
I would be happy if you can make further contributions 👌 thank you

**Comment by AnthonyNahas** [comment]
@kuncevic this issue as feature will be merged with #76 in the next release

**Comment by AnthonyNahas** [comment]
@kuncevic any update ? I would like to close this issue since it's not directly related to the library..

Please consider reopening the ticket if you have any idea or suggestion to implement

thanks for contributing
